docs/_articles/en/getting-started/recommended-extensions.md

```diff
@@ -19,23 +19,20 @@ Keeping your code consistent and nicely formatted is important in any programmin
 
 > NOTE: Prettier also supports Apex formatting through the [Prettier Apex plugin](https://github.com/dangmai/prettier-plugin-apex).
 
-## XML
+## ESLint
 
-VS Code doesn’t ship with rich XML tools by default. This extension, built by Red Hat, provides language support for for XML documents such as the `-meta.xml` or `package.xml` files in your VS Code project.
+Analyzes your JavaScript code to find issues and helps you fix them. As a part of the LWC extension, Salesforce provides specific ESLint rules and you can also add your own.  These preconfigured rules to help you write high quality code.  
 
-<https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml>
+<https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint>
 
-## GitHub Pull Requests
 
-This extension makes working with GitHub even easier.
+## XML
 
-<https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github>
+VS Code doesn’t ship with rich XML tools by default. This extension, built by Red Hat, provides language support for for XML documents such as the `-meta.xml` or `package.xml` files in your VS Code project.
```

Comment:
with GitLens, we're providing the link. Do you not want to do that with the XML tool? And is there an official name for it?